src/Storages/StorageInMemoryMetadata.h

```diff
@@ -245,4 +245,29 @@ struct StorageInMemoryMetadata
 using StorageMetadataPtr = std::shared_ptr<const StorageInMemoryMetadata>;
 using MultiVersionStorageMetadataPtr = MultiVersion<StorageInMemoryMetadata>;
 
+/// Daisy: starts.
+/// Common CreateQuery for all storages. Contains all possible parts of CREATE
+/// query from all storages.
+struct StorageInMemoryCreateQuery
+{
+    String query;  /// if show_table_uuid_in_table_create_query_if_not_nil = 0
+    String query_uuid;  /// if show_table_uuid_in_table_create_query_if_not_nil = 1
+    String engine_full;
+
+    StorageInMemoryCreateQuery() = default;
```

Comment:
can we remove all of the default constructors or assignments and remove all the setter / getter functions ?